ckanext/cnra_schema/templates/scheming/package/snippets/package_form.html

```diff
@@ -1,9 +1,53 @@
 {% ckan_extends %}
 
+{% block basic_fields %}
+  {% block styles %}
+      <link rel="stylesheet" href="/css/additional_metadata_form.css" />
+  {% endblock%}
+
+  <p class=right-text>
+    Questions? See: <a href="/pages/metadata"><u>Metadta Guide</u></a>
```

Comment:
I think you should remove the colon from `Questions? See: `.
Also `Metadta Guide` should be `Metadata Guide`.

Reply:
CNRA requested that the text say "Questions? See: Metadata Guide" 
<img width="797" alt="Screen Shot 2020-06-17 at 1 50 26 PM" src="https://user-images.githubusercontent.com/49450112/84945722-98f53f80-b0b5-11ea-916a-50e2b349d59f.png">